core/vm/message/impl/message_signer_impl.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_MESSAGE_SIGNER_IMPL_HPP
+#define CPP_FILECOIN_CORE_VM_MESSAGE_SIGNER_IMPL_HPP
+
+#include "common/logger.hpp"
+#include "storage/keystore/keystore.hpp"
+#include "vm/message/message_signer.hpp"
+
+namespace fc::vm::message {
+
+  using storage::keystore::KeyStore;
+
+  /**
+   * Class MessageSignerImpl.
+   */
+  class MessageSignerImpl : public MessageSigner {
+   public:
+    MessageSignerImpl() = delete;
```

Comment:
I think there is no sense to delete default constructor explicitly since implicitly-default constructor is created only if no user-defined constructors are provided. But there is the user-defined constructor on the next line, so it is not the case.